File: Kotlin(android_app)/app/java/jewelrystore/models/ProductsWithType.kt
package com.sysoliatina.jewelrystore.models

import com.google.gson.annotations.SerializedName

data class ProductsWithType(
    @SerializedName("product_type_id") val productTypeId: Int,
    @SerializedName("product_type") val productType: String,
    val products: List<Product>
)

File: Python(create_db)/models/products_with_type.py
from typing import List

from pydantic import BaseModel
from sqlite3 import Date
from models import tables
from models.product import ProductWithWish


class ProductsWithType(BaseModel):
    product_type_id: int
    product_type: str
    products: List[ProductWithWish] = []

    class Config:
        orm_mode = True
